atomate/vasp/database.py

```diff
@@ -227,6 +182,12 @@ def retrieve_task(self, task_id):
             calc["aeccar2"] = aeccar["aeccar2"]
         return task_doc
 
+    def insert_object(self, use_gridfs, *args, **kwargs):
+        if self._maggma_store_type is not None:
+            return self.insert_maggma_store(*args, **kwargs)
+        elif use_gridfs:
+            return self.insert_gridfs(*args, **kwargs)
```

Comment:
What should this do if use_gridfs is False and `self._maggma_store_type is not None`? What did it do before? Should it try and add it to mongo anyway?

Reply:
It will try to add to mongo directly, which should work for some smaller documents.  But will throw an error for larger documents

Reply:
Ok great. Just checking! Thanks.